### ISSUE TITLE
What purpose does the Export directory serve?

### ISSUE THREAD

**Issue by p-baum** [photoprism/photoprism#276]
I noticed there was once an export command but that is not available on my docker installation.  Searching the docs and github reveals no answers. What is the Export directory for?

I would like it if my import masters were dumped there after import.

If thats not what its for then I have a related question:

How do I get my originals back out?  (AKA vendor lock-in.)

my originals = import masters
your originals in "Originals" are not my originals in "Import". 
Naming scheme is confusing.

**Comment by lastzero** [comment]
We currently use the export directory to create zip files for downloading. You can backup originals however you like, it's your photos in whatever format they are. We don't modify them, only rename them if you import them to avoid duplicates.

**Comment by p-baum** [comment]
And when and how do 'you' do that?  I just checked my Export directory and its still empty.

**Comment by lastzero** [comment]
Select files and click on download in the UI (context menu, bottom right).

**Comment by lastzero** [comment]
Note that zip files might be deleted once download is complete.

**Comment by p-baum** [comment]
OK. Thankyou.

I will raise another issue regarding real export & vendor-lockin

**Comment by lastzero** [comment]
Just removed the export path to avoid confusion. Replaced it with temp path where needed.
